src/components/TextInput/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { InpuText, LabelText } from './styles';
+const TextInput = ({ label, id, type, logical, placeholder }) => {
+  return (
+    <>
```

Comment:
It would be better to encapsulate the label and the input inside a `<div>` or a `<fieldset>` so they would be easier to manipulate as a whole. That way we can skip the use of the `<br>` and set the `margins` and positions of both elements at the same time.
<img width="306" alt="Captura de Pantalla 2020-08-25 a la(s) 11 49 20 p  m" src="https://user-images.githubusercontent.com/17033709/91256690-9d7d4a80-e72d-11ea-8ed0-be93d7757c55.png">

Reply:
mmmm the Idea was this component will be used on another like Form, and on this manipulate the component for example 
```
import { useFormFields, useFormTextField } from '../hooks/useFormFields';
import TimerLabel from '../components/TextInput';

function LoginForm() {
  const emailField = useFormField();
  const passwordField = useFormField();

  const handleSubmit = (e: React.FormEvent) => {
    e.preventDefault();
    api.login(emailField.value, passwordField.value);
  };
  return (
    <form onSubmit={handleSubmit}>
      <div>
          <TextInput label='User Email' logical={emailField}  type={'email'} id={'userEmail'} placeholder={''} />
      </div>
       <div>
          <TextInput label='User Password' logical={passwordField}  type={'password'} id={'userPw'} placeholder={'xxxxxx'} />
      </div>`
```